src/main/steps/applicant1/finalising-your-application/content.ts

```diff
@@ -33,8 +35,35 @@ const en = ({ isDivorce, partner, userCase }: CommonContent) => ({
   },
 });
 
-// @TODO translations
-const cy: typeof en = en;
+const cy: typeof en = ({ isDivorce, partner, userCase }: CommonContent) => ({
+  title: `Ydych chi eisiau ${isDivorce ? 'cadarnhau eich ysgariad' : "dod â'ch partneriaeth sifil i ben"}?`,
+  line1: `Bydd eich ${
+    isDivorce ? 'priodas' : 'partneriaeth sifil'
+  } yn dod i ben yn gyfreithiol pan wneir y gorchymyn terfynol. Gallai hyn effeithio ar eich sefyllfa ariannol.`,
+  warningText: `Os nad ydych wedi gorffen cynnal trafodaethau am eich arian, eiddo neu asedau eraill yna dylech ofyn am gyngor cyfreithiol cyn
+  ${isDivorce ? 'cadarnhau eich ysgariad' : 'dod â’ch partneriaeth sifil i ben'}.`,
+  readMore: {
+    subHeader: `Os ydych eisiau sortio eich sefyllfa ariannol cyn ${
+      isDivorce ? 'cadarnhau eich ysgariad' : 'dod â’ch partneriaeth sifil i ben'
+    }`,
+    line1: 'Dylech gadw’r cais ac allgofnodi, a sortio eich sefyllfa ariannol cyn gwneud cais am orchymyn terfynol.',
+    line2: `Os nad ydych wedi gwneud cais erbyn ${getFormattedDate(
+      userCase.dateFinalOrderEligibleToRespondent,
+      SupportedLanguages.Cy
+    )} yna bydd eich ${partner} yn gallu gwneud cais.
+    Mae’n bosib y bydd yn rhaid i’r ddau ohonoch ddod i wrandawiad llys, os byddant yn gwneud cais.`,
+    line3:
+      "Os byddwch yn disgwyl blwyddyn cyn gwneud cais yna bydd angen i chi egluro’r rhesymau dros yr oedi i'r llys.",
+  },
+  checkboxLine: `Rwyf eisiau ${isDivorce ? 'cadarnhau fy ysgariad' : "dod â'm partneriaeth sifil i ben"}`,
+  continue: 'Cyflwyno',
+  errors: {
+    doesApplicant1WantToApplyForFinalOrder: {
+      required:
+        'Ni allwch barhau heb ddewis opsiwn o’r blwch ticio. Os nad ydych eisiau parhau yna dylech gadw’r cais ac allgofnodi.',
```

Comment:
this might also be avaliable in the common content file.

Reply:
This is not present in common.